crates/red_knot_module_resolver/src/typeshed/versions.rs

```diff
@@ -3,10 +3,20 @@ use std::fmt;
 use std::num::{NonZeroU16, NonZeroUsize};
 use std::ops::{RangeFrom, RangeInclusive};
 use std::str::FromStr;
+use std::sync::Arc;
 
+use ruff_db::{source::source_text, vfs::VfsFile};
 use rustc_hash::FxHashMap;
 
-use crate::module::ModuleName;
+use crate::db::Db;
+use crate::module_name::ModuleName;
+use crate::supported_py_version::SupportedPyVersion;
+
+#[salsa::tracked]
```

Comment:
You can use `return_ref` to avoid the need for an `Arc`
```suggestion
#[salsa::tracked(return_ref)]
```

Reply:
Ah nice, this means I also don't need to derive `Clone` at all for `TypeshedVersions`! I didn't like that at all.